README.md

```diff
@@ -50,21 +50,36 @@ Probably you should read Projectile's [README](https://github.com/bbatsov/projec
 
 ### Customizing
 
+#### Keymap prefix
+Prior to version 0.20.0 the keymap prefix used to be setup. Following the [convention](https://www.gnu.org/software/emacs/manual/html_node/elisp/Key-Binding-Conventions.html) we have stopped doing this automatically. In order to get the old behaviour you need to do:
+```el
+(setq projectile-rails-keymap-prefix (kbd "C-c r"))
+```
```

Comment:
In projectile they recommand `(define-key projectile-mode-map (kbd "C-c p") 'projectile-command-map)`, which makes sense according to the `make-obsolete-variable` you used below.

The idea is to get rid of `*-prefix` variables altogether.

Reply:
Similar to projectile I've left the prefix variable, but it is not `nil` and marked as obsolete. I've also recommended:
```el
(define-key projectile-rails-mode-map (kbd "C-c r") 'projectile-rails-command-map)
```
instead of:
```el
(setq projectile-rails-keymap-prefix (kbd "C-c r"))
```